### DISCUSSION TITLE
Isomorphic Crystal, a framework using Blueprint (and Webview)

### DISCUSSION THREAD

**Discussion by serge-hulne** [stephannv/blueprint#50] [Show and tell]
https://github.com/serge-hulne/isomorphic-crystal-view

GUI lib for Crystal-lang allowing for developing simple desktop apps using Crystal only.

- Expressive (JSX-like).
- Concise (it's like writing HTML).
- Modular (A custom visual widget can include other custom widgets)
- Stylable using regular CSS.
- Easy layout coding.
- Easy state management of the app.

**Comment by stephannv** [comment]
Nice job @serge-hulne . Thanks for this. It's the first project using Blueprint. I will have to create a section listing all projects that are using blueprint.

**Reply by serge-hulne** [reply]
Thank you very much.
Please let me know if you have any suggestions, remarks, etc...
